test/scons-time/run/config/python.py

```diff
@@ -46,14 +46,13 @@
 test.write(
     'config',
     f"""\
-python = f'{my_python_py}'
+python = r'{my_python_py}'
```

Comment:
does this need to be `fr` as well?

Reply:
No.  This stuff is still hard to think about with code-inside-a-string, and that's the mistake I made: the braces are for interpolation in the big triple-quoted string; by the time the python statement in the created file is interpreted there's nothing left to interpolate - but the string in the assignment contains a (misidentified) Unicode escape, so has to be a rawstring.